services/src/main/java/org/exoplatform/news/notification/plugin/PublishNewsNotificationPlugin.java

```diff
@@ -0,0 +1,71 @@
+package org.exoplatform.news.notification.plugin;
+
+import org.exoplatform.commons.api.notification.NotificationContext;
+import org.exoplatform.commons.api.notification.model.NotificationInfo;
+import org.exoplatform.commons.api.notification.plugin.BaseNotificationPlugin;
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.news.notification.utils.NotificationConstants;
+import org.exoplatform.news.notification.utils.NotificationUtils;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+
+public class PublishNewsNotificationPlugin extends BaseNotificationPlugin {
+
+  private static final Log   LOG = ExoLogger.getLogger(PublishNewsNotificationPlugin.class);
+
+  public static final String ID  = "PublishNewsNotificationPlugin";
+
+  public PublishNewsNotificationPlugin(InitParams initParams) {
+    super(initParams);
+  }
+
+  @Override
+  public String getId() {
+    return ID;
+  }
+
+  @Override
+  public boolean isValid(NotificationContext ctx) {
+    return true;
+  }
+
+  @Override
+  protected NotificationInfo makeNotification(NotificationContext ctx) {
+    String contentTitle = ctx.value(PostNewsNotificationPlugin.CONTENT_TITLE);
```

Comment:
I think that it is better to define and use it own static final attributes CONTENT_TITLE .... not those of PostNewsNotificationPlugin